sktime/utils/_testing/forecasting.py

```diff
@@ -30,12 +30,12 @@ def _get_n_columns(tag):
     elif tag == "both":
         n_columns_list = [1, 2]
     else:
-        raise ValueError(f"Unexpected tag {tag}.")
+        n_columns_list = [1]
```

Comment:
why are you changing this? The `ValueError` is more sensible, returning a list no matter what is just inviting errors, e.g., in case of a typo.

Reply:
bcz not all estimators have the tag `scitype:y` but we still need to give `n_columns`